+nigeLab/@Sort/parseBlocks.m

```diff
@@ -32,7 +32,8 @@
    
    % Check the format of files
    fprintf(1,'\nChecking SORTED for %s...000%%\n',blockObj(ii).Name);
-   curCh = 0; nCh = numel(blockObj(ii).Mask);
+   curCh = 0;
+   nCh = numel(blockObj(ii).Mask);
    for iCh = blockObj(ii).Mask
       blockObj(ii).checkSpikeFile(blockObj(ii).Mask);
```

Comment:
Wait, for every channel it checks the spike file using the Mask, but how does it know what the current channel is?

Reply:
I actually did not touch this, I only added the two things at the end. You're right though. Both curCh and iCh are not used. It was taking insanely long to do the checks, maybe that's why. let me look into it

Reply:
Now should be better.